mne/time_frequency/tests/test_tfr.py

```diff
@@ -1218,25 +1218,29 @@ def test_averaging_freqsandtimes_epochsTFR():
         avgpower = power.average(method=lambda x: np.mean(x, axis=2), **kwargs)
 
 
-@pytest.mark.parametrize("n_drop", (0, 2))
-def test_epochstfr_getitem(epochs_full, n_drop):
+@pytest.mark.parametrize("n_drop, as_array", ((0, False), (0, True), (2, False)))
```

Comment:
can we change this to `as_tfr_array` or even `as_tfrarray`?  `as_array` has strong associations with https://numpy.org/doc/stable/reference/generated/numpy.asarray.html so I went down the wrong mental path reading this diff.

Reply:
Have updated this now.